setup.py

```diff
@@ -62,11 +62,6 @@ def get_data_files():
     data_files = []
     data_files += [(get_tests_dir(), glob.glob('examples/tests/*.py'))]
     data_files += [(get_tests_dir(), glob.glob('examples/tests/*.sh'))]
-    for data_dir in glob.glob('examples/tests/*.data'):
```

Comment:
I missed this change. Wouldn't it make sense to have at least some example `.data` in our example `tests`? I mean the `example` tests are only useful as examples and not having `.data` in examples is IMO a big drawback.

Reply:
Oh, it looks like I mistakenly read the path.  I read it as:

```
examples/tests/.data*
```

Instead of what's really in there.  So, this commit should be removed.